protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js

```diff
@@ -97,5 +97,11 @@ return network.registerProtocol('qmi', {
 		o = s.taboption('advanced', form.Value, 'mtu', _('Override MTU'));
 		o.placeholder = dev ? (dev.getMTU() || '1500') : '1500';
 		o.datatype    = 'max(9200)';
+		
+		o = s.taboption('general', form.ListValue, 'pdptype', _('PDP Type'));
+		o.value('ipv4v6', 'IPv4/IPv6 (both)');
```

Comment:
Could you please translate this? The value **(both)** is not common for all language.

Reply:
Good point, I guess I'll get line 71 as well?  I based it on that format.  Will commit tomorrow

Reply:
I would say omit "both" because I think it is self-explanatory

Reply:
Done, but only for the IPv4/IPv6 value